src/PrestaShopBundle/Controller/Admin/Improve/International/StateController.php

```diff
@@ -58,10 +70,173 @@ public function getStatesAction(Request $request)
             ]);
         } catch (Exception $e) {
             return $this->json([
-                    'message' => $this->getErrorMessageForException($e, []),
-                ],
+                'message' => $this->getErrorMessageForException($e, []),
```

Comment:
You don't define any custom messages, is that intentional?

Reply:
Well i only update code style here but i guess it was intended for some reason.